google-cloud-clients/google-cloud-translate/src/main/java/com/google/cloud/translate/Translate.java

```diff
@@ -116,15 +116,35 @@ public static TranslateOption format(String format) {
    *
    * <p>Example of listing supported languages, localized according to
    * {@link TranslateOptions#getTargetLanguage()}:
-   * <pre> {@code
-   * List<Language> languages = translate.listSupportedLanguages();
-   * }</pre>
+   * <!--SNIPPET translate_list_codes-->
+   * <pre>{@code
+   *    // TODO(developer): Uncomment these lines.
+   *    // import com.google.cloud.translate.*;
```

Comment:
Are we using `*` here to keep things shorter as this is more of an example as part of the library?

Reply:
This example appears in the JavaDoc for the library. `*` It is included here by the `snippets.go` utility.